api_spec.yaml

```diff
@@ -3736,6 +4247,119 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
+  /recipe/multitenancy/tenant/user:
+    post:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Add user's association with the tenant.
+
+        Note: To associate a user to a tenant, the tenant must be in the same user pool
+        and user must already exist in the app
+
```

Comment:
the API for getUser in all the recipes will also return the list of tenants for that user.